src/components/esn-app-grid/icons/AdminAppIcon.tsx

```diff
@@ -0,0 +1,36 @@
8l0.11,-0.049l0.109,-0.05l0.109,-0.052l0.107,-0.052l0.107,-0.054l0.107,-0.056l0.105,-0.056l0.105,-0.058l0.104,-0.059l0.104,-0.061l0.102,-0.061l0.102,-0.063l0.101,-0.063l0.1,-0.065l0.099,-0.066l0.099,-0.068l0.098,-0.069l0.096,-0.069l0.096,-0.071l0.095,-0.072l0.094,-0.073l0.093,-0.074l0.093,-0.075l0.091,-0.077l0.09,-0.077l0.09,-0.079l0.088,-0.079l0.088,-0.081l0.087,-0.081l0.085,-0.083l0.085,-0.084l0.083,-0.084l0.083,-0.086l0.082,-0.087l0.081,-0.087l0.079,-0.089l0.078,-0.089l0.078,-0.091l0.076,-0.091l0.076,-0.092l0.074,-0.094l0.073,-0.094l0.072,-0.095l0.07,-0.096l0.07,-0.096l0.069,-0.098l0.067,-0.098l0.066,-0.1l0.065,-0.1l0.064,-0.101l0.063,-0.101l0.061,-0.103l0.061,-0.103l0.058,-0.104l0.058,-0.105l0.057,-0.106l0.055,-0.106l0.054,-0.107l0.053,-0.108l0.051,-0.109l0.051,-0.109l0.048,-0.11l0.048,-0.111l0.047,-0.111l0.044,-0.112l0.044,-0.112l0.042,-0.114l0.041,-0.114l0.04,-0.114l0.038,-0.115l0.037,-0.116l0.035,-0.117l0.034,-0.117l0.033,-0.118l0.031,-0.118l0.03,-0.119l0.028,-0.119l0.027,-0.12l0.026,-0.121l0.024,-0.121l0.022,-0.121l0.021,-0.123l0.02,-0.122l0.018,-0.123l0.016,-0.124l0.015,-0.125l0.014,-0.124l0.012,-0.125l0.01,-0.126l0.009,-0.126l0.007,-0.127l0.006,-0.127l0.004,-0.128l0.002,-0.128l0.001,-0.128l-0.001,-0.129l-0.002,-0.128l-0.004,-0.128l-0.006,-0.127l-0.007,-0.127l-0.009,-0.126l-0.01,-0.126l-0.012,-0.125l-0.014,-0.124l-0.015,-0.125l-0.016,-0.124l-0.018,-0.123l-0.02,-0.122l-0.021,-0.123l-0.022,-0.121l-0.024,-0.121l-0.026,-0.121l-0.027,-0.12l-0.028,-0.119l-0.03,-0.119l-0.031,-0.118l-0.033,-0.118l-0.034,-0.117l-0.035,-0.116l-0.037,-0.116l-0.038,-0.115l-0.04,-0.115l-0.041,-0.114l-0.042,-0.113l-0.044,-0.113l-0.045,-0.112l-0.046,-0.111l-0.048,-0.111l-0.048,-0.11l-0.051,-0.109l-0.051,-0.109l-0.053,-0.107l-0.054,-0.108l-0.055,-0.106l-0.057,-0.105l-0.058,-0.106l-0.059,-0.103l-0.06,-0.104l-0.061,-0.102l-0.063,-0.102l-0.064,-0.101l-0.065,-0.1l-0.066,-0.099l-0.067,-0.099l-0.069,-0.097l-0.07,-0.097l-0.07,-0.096l-0.072,-0.095l-0.073,-0.094l-0.074,-0.093l-0.076,-0.093l-0.076,-0.091l-0.078,-0.091l-0.078,-0.089l-0.079,-0.088l-0.081,-0.088l-0.082,-0.087l-0.082,-0.085l-0.084,-0.085l-0.085,-0.084l-0.085,-0.082l-0.087,-0.082l-0.088,-0.081l-0.088,-0.079l-0.09,-0.079l-0.09,-0.077l-0.091,-0.076l-0.093,-0.076l-0.093,-0.074l-0.094,-0.073l-0.095,-0.072l-0.096,-0.07l-0.096,-0.07l-0.098,-0.069l-0.099,-0.067l-0.099,-0.066l-0.1,-0.065l-0.101,-0.064l-0.102,-0.063l-0.102,-0.061l-0.104,-0.06l-0.104,-0.06l-0.105,-0.057l-0.105,-0.057l-0.107,-0.055l-0.107,-0.054l-0.108,-0.053l-0.108,-0.051l-0.109,-0.051l-0.11,-0.049l-0.111,-0.047l-0.111,-0.047l-0.112,-0.044l-0.113,-0.044l-0.113,-0.042l-0.114,-0.041l-0.115,-0.04l-0.115,-0.038l-0.116,-0.037l-0.116,-0.035l-0.117,-0.034l-0.118,-0.033l-0.118,-0.031l-0.119,-0.03l-0.12,-0.028l-0.119,-0.027l-0.121,-0.026l-0.121,-0.024l-0.122,-0.022l-0.122,-0.021l-0.123,-0.02l-0.123,-0.018l-0.123,-0.016l-0.125,-0.015l-0.124,-0.014l-0.126,-0.012l-0.125,-0.01l-0.127,-0.009l-0.126,-0.007l-0.128,-0.006l-0.127,-0.004l-0.128,-0.002l-0.129,-0.001l-0.128,0.001Z"
```

Comment:
Can we store these `d` values in files or constants? They make this file is hard to read.

Reply:
Doing that can make the SVG component files fragmented and unnecessarily more complex. SVG files are often hard to read anyway.